RoboJackets/robocup-software
# Split wall tactic

## Description
Wall tactic should only take one robot per tactic.

## Associated Issue
Issue #???

## Design Documents
n/a

## Steps to test
### Test Case 1
1. make
2. run sim
3. switch to test play selector and choose a play that uses wall tactic
    plays that use wall: basic_122, basic_defense, defend_restart, kickoff_play, restart, wall_ball
4. run sim again

Expected result: should still work
